README.md

```diff
@@ -2,38 +2,33 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-auditor-iam/workflows/build/badge.svg)](https://github.com/cisagov/cool-auditor-iam/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This project is used to manage IAM permissions for auditor users.
 
-## Usage ##
+## Pre-Requisites ##
```

Comment:
This should be "Pre-requisites", in keeping with our style giude's suggestion to use sentence case for all headings below the top level:
```suggestion
## Pre-requisites ##
```